consensus/istanbul/backend/backend.go

```diff
@@ -202,12 +202,7 @@ func (sb *backend) Gossip(valSet istanbul.ValidatorSet, payload []byte) error {
 
 // checkInSubList checks if the node is in a sublist
 func (sb *backend) checkInSubList(prevHash common.Hash, valSet istanbul.ValidatorSet) bool {
```

Comment:
```suggestion
func (sb *backend) checkInCommittee(prevHash common.Hash, valSet istanbul.ValidatorSet) bool {
```

Reply:
"SubList" is commonly used in overall code instead of "Committee".
I will consider changing all of them later.

Reply:
what about remove `func (sb *backend) checkInSubList` and use valSet.CheckInSubList directly on line 243?

Reply:
@laggu Good suggestion. I will consider it later.